release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -20,6 +20,8 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 {product-title} {product-version} is based on {op-system-base-full} 9.2. {op-system-base} 9.2 has not yet been submitted for FIPS validation. Red Hat expects, though cannot commit to a specific timeframe, to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Updates will be available in link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards].
 
+Because {product-title} 4.13 is based on the RHEL 9.2 host operating system, the microarchtecture requirements are now raised to x86-64-v2, Power9, and Z14. Please see the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL Microarchtecture requirements documentation]. You may verify compatibility by following the procedures outlined in this link:https://access.redhat.com/solutions/7052996[KCS article].
```

Comment:
```suggestion
Because {product-title} 4.13 is based on the RHEL 9.2 host operating system, the microarchtecture requirements are now raised to x86-64-v2, Power9, and Z14. Please see the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL Microarchtecture requirements documentation]. You may verify compatibility before updating by following the procedures outlined in this link:https://access.redhat.com/solutions/7052996[KCS article].
```